src/main/java/domain/Number.java

```diff
@@ -1,11 +1,14 @@
 package domain;
 
-import java.util.Objects;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
-public class Number {
+public class Number implements Comparable<Number> {
+    private static final Random random = new Random();
     private static final int LOTTO_NUMBER_MIN = 1;
     private static final int LOTTO_NUMBER_MAX = 45;
-
+    private static final List<Number> cached = IntStream.rangeClosed(1, 45).boxed().map(Number::new).collect(Collectors.toList());
```

Comment:
```suggestion
    private static final List<Number> cached = IntStream.rangeClosed(1, 45)
    .boxed()
    .map(Number::new)
    .collect(Collectors.toList());
```
스트림은 기능단위로 줄바꿈 하여 가독성을 높여주세요.